packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -98,6 +75,11 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
   }
 
   public render(): JSX.Element {
+    if (this._previousProps !== this.props) {
```

Comment:
Not so sure about this approach... I know you were concerned about the perf impacts of using derived state, but moving to something that's generally considered an anti-pattern (side effects during render) is probably not the right solution.